lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -162,6 +167,72 @@ function determineNaturalSize(url) {
   });
 }
 
+/**
+ * @param {LH.Crdp.CSS.CSSStyle} [style]
+ * @param {string} property
+ * @return {boolean}
+ */
+function hasSizeDeclaration(style, property) {
+  return !!style && !!style.cssProperties.find(({name}) => name === property);
```

Comment:
you can remove all the bangs if you change `find` to `some`.

Reply:
sounds good! although I'm leaning towards implementing the version @patrickhulce recommended:

> `if (hasSizeDeclaration(inlineStyle, property)) {`
> consider flipping the above fn to findSizeDeclaration and the if to checking if it's defined? then we don't need ts-> > ignores and we don't .find twice :)

because with `some` I'd still have to use `find` again after that